tools/proto_format/proto_format.sh

```diff
@@ -21,7 +21,7 @@ then
   bazel test ${BAZEL_BUILD_OPTIONS} //tools/protoxform:merge_active_shadow_test
 fi
 
-# Generate //versioning:active_protos.
+# Generate //versioning:active_protos & versioning::frozen_protos.
```

Comment:
Sorry, what is `versioning::frozen_protos`? Are you referring to this bazel target here: https://github.com/envoyproxy/envoy/blob/98d2f3b553b87c3e935f57ba15b4faf68b45d7f0/api/versioning/BUILD#L168?

Reply:
yes.

Reply:
I’m not sure writing it as “versioning::frozen_protos” is correct here.

Reply:
Oh sorry for that !!!  so you want to correct statement ?

Reply:
I think you should use "and" and the `bazel` target, instead of "versioning::frozen_protos". I'm not really sure what `::` stands for.

Reply:
@dio thanks for reply. is this ok now ? 
# Generate //versioning:active_protos and versioning:frozen_protos

Reply:
one quick think how to mention bazel target. like in url it is https://github.com/envoyproxy/envoy/blob/98d2f3b553b87c3e935f57ba15b4faf68b45d7f0/api/versioning/BUILD#L168.